compiler/rustc_target/src/spec/targets/riscv32imafc_unknown_none_elf.rs

```diff
@@ -0,0 +1,23 @@
+use crate::spec::{Cc, LinkerFlavor, Lld, PanicStrategy, RelocModel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        data_layout: "e-m:e-p:32:32-i64:64-n32-S128".into(),
+        llvm_target: "riscv32".into(),
+        pointer_width: 32,
+        arch: "riscv32".into(),
+
+        options: TargetOptions {
+            linker_flavor: LinkerFlavor::Gnu(Cc::No, Lld::Yes),
+            linker: Some("rust-lld".into()),
```

Comment:
You need `llvm_abiname: "ilp32f".into(),`, right? Otherwise the soft-float abi would be chosen by rustc and you would end up with a linker error.

Reply:
Thank you!